src/components/logs/LinePart.tsx

```diff
@@ -9,12 +11,37 @@ interface Props {
 function LinePart({ parsedLine, lastPart }: Props) {
     const StyledLogLinePart = styled('span')(parsedLine.css);
 
+    const splitTextLines = parsedLine.text.split('\\n');
+
+    const linePartRendered = useMemo(() => {
+        return (
+            <>
+                {splitTextLines.map((lineText, index) => {
+                    const formattedLine = unescapeString(
+                        lastPart ? lineText.trimEnd() : lineText
+                    );
+
+                    return (
+                        <ListItemText
+                            key={`${lineText} - linePart - ${index}`}
+                            primary={formattedLine}
+                            disableTypography
+                        />
+                    );
+                })}
```

Comment:
eventually this should print out JSON objects nicely - that is why there is code that is more verbose than needed.